CustomerApp/src/main/java/com/brainscratch/laboratory/customerapp/Json.java

```diff
@@ -0,0 +1,45 @@
+package com.brainscratch.laboratory.customerapp;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+
+public class Json {
+    public static void main(String[] args) {
+
+        ObjectMapper mapper = new ObjectMapper();
+
+        Customer customer = createCustomer();
+
+        try {
+
+            // Java objects to JSON file
+            mapper.writeValue(new File("c:\\test\\customer.json"), customer);
```

Comment:
You forgot that customer is not alone: there are 0..infinity customers in the app.

Reply:
so i have to create a counter or something?